app/controllers/api/v1/facts_controller.rb

```diff
@@ -0,0 +1,55 @@
+class Api::V1::FactsController < ApplicationController
+  include AuthenticationCheck
+
+  before_action :is_user_logged_in
+  before_action :set_fact, only: [:show, :update, :destroy]
+
+  # GET /members/:member_id/facts
+  def index
+    @member = Member.find(params[:member_id])
+    render json: @member.facts # note that because the facts route is nested inside members
+                             # we return only the facts belonging to that member
+  end
+
+  # GET /members/:member_id/facts/:id
+  def show
+    render json: @fact, status: 200
```

Comment:
I'm not sure if this is covered in your authorization somewhere, but I couldn't find it. Seems like you're meant to also check to see if the member has access to this fact as well. It's a security concern if they can enter any fact id they want to see.